src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -141,17 +141,37 @@ Okta, Azure, and OneLogin have New Relic apps for both the original user model a
 
 ![User migration page 5](./images/user-migration-page-5.png "user-migration-page-5.png")
 
-**Recommended**: Download the full list of existing original user model users before choosing to import users. This will be a useful resource and serve as a backup, if you need it. 
+There are two methods for adding and managing your New Relic users. Select the method you'll be using for instructions and tips: 
 
-After downloading your original user model users, you can upload all users or just some of them. We recommend reviewing this list of users and removing any users you don't want migrated (for example, people who no longer work at your organization). 
+<CollapserGroup>
+  <Collapser
+    id="manual-provision"
+    title="Add and manage users from the New Relic UI"
+  >
+**Recommended**: Download the full list of existing original user model users before choosing to import users. This will be a useful resource and serve as a backup, if you need it. You can also use this list to help you figure out which users have access to which accounts, and also help you create groups and access grants during a later step.
+    
+During this migration we provide a capability to upload users in bulk to make the task of adding users manually much easier. After downloading your original user model users, you can upload all users or just some of them. We recommend reviewing this list of users and removing any users you don't want migrated (for example, people who no longer work at your organization) from the list provided as well as the [New Relic UI for original user model.](/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model#deleting) 
 
 When you reupload your list of users and complete this step, it will create user records on the New Relic One user model. In a later step, you’ll be able to transition these users' assets. 
 
-The new user records created for your users have the same login credentials: there's no need to reset passwords. If a user has a pending email verification status (pending being verified), that will also be transitioned over. 
-
-Tips and considerations: 
+Some tips and considerations: 
+* If using username/password for your users' login, they'll have the same login credentials on the new user record created: there's no need to reset passwords. If a user has a pending email verification status (pending being verified), that will also be transitioned over. 
```

Comment:
The "If" sentence reads easier without the colon, then having "There's no need to reset passwords" be a punchier sentence. 

Also, might clarify at what point they will be transitioned over. Is there a time frame between verification and the transition? Is it immediate?